components/typography/Base/index.tsx

```diff
@@ -487,7 +487,7 @@ const Base = React.forwardRef<HTMLElement, BlockProps>((props, ref) => {
               width={ellipsisWidth}
               onEllipsis={onJsEllipsis}
               expanded={expanded}
-              miscDeps={[copied, expanded, copyLoading]}
+              miscDeps={[copied, expanded, copyLoading, copyConfig.text]}
```

Comment:
`miscDeps` is used for trigger measure ellipsis update. This will make useless re-render. I patched in onCopy side, pls help to check: https://github.com/ant-design/ant-design/pull/48347